aips/aip-31.md

```diff
@@ -20,7 +20,15 @@ This AIP proposes to add a flag in `delegation_pool.move` to allow “permission
 
 Some delegation pools may require a KYC process before accepting delegated stake for compliance reasons. Since delegation pools are permissionless, this feature gives the pool owner more control over who the delegation pool can accept stake from. 
 
-## Rationale
+## Specification
+
+Any account can construct their own allowlist of addresses, independent of the delegation pool, by interacting with the `aptos_framework::delegation_pool_allowlist` module.
```

Comment:
is there a way to check if allowlisting is enabled?

Reply:
Given a delegation pool of address `pool_address`, one can check if allowlisting is enabled by:

1. checking if `delegation_pool::ownership_lookup_enabled(pool_address)` is true

2. checking if `delegation_pool_allowlist::allowlisting_enabled(delegation_pool::get_pool_owner(pool_address))` is true

`delegation_pool` exposes `delegator_allowlisted` which already checks whether ownership lookup is enabled + accesses the allowlist defined on the pool owner.
We can also expose `allowlisting_enabled` on `delegation_pool` to merge steps 1 and 2 from above.